src/processTranslation.tsx

```diff
@@ -60,14 +62,25 @@ export const processTranslation = ({
     throw new Error(`Missing translation: "${key}" with suffix: "${suffix}"`);
   }
 
+  const {
+    context,
+    ns,
+    ...replaceableArgs
+  } = args;
+
   // Replace placeholders like `{{count}}` with values from `args`
-  const processed = Object.entries(args).reduce(
-    (v, [ name, value ]) => v.replace(
-      new RegExp(`{{\\s*${name}\\s*}}`, 'g'),
-      String(formatArgument(locale, value)),
-    ),
+  return Object.entries(replaceableArgs).reduce(
+    (v, [ name, value ]) => {
+      const regexp = new RegExp(`{{\\s*${name}\\s*}}`, 'g');
+
+      if (onError && !regexp.test(v)) {
+        onError(`Argument: "${name}" with value: "${value}" is not valid`);
```

Comment:
I think we need a better API for error handling:
- If you just pass in this string to the error handler, then the consumer needs to parse the string to get the `name` and `value`, etc.
- With the current implementation there is no way to provide a fallback text / value when this happens - e.g. if a key does not exist then instead of returning the key name the consumer can't return a custom text to be displayed.

Reply:
- Added the second param with target localized string key, argument name and value that failed to be replaced

- It is not intended to provide fallback, the main reason for this PR was to track this. 

The initial case was the wrong argument specified for DE locale and we were not able to replace it with the key/value pair used in the code.

Reply:
> It is not intended to provide fallback
why? it would be really easy to support this

Reply:
You have the localized string: `"stringWithParam": "text with parameter: {{wrongParam}}"`
and you are executing something like in your code: `t('stringWithParam', { param: "param value" })`
The current outcome would be: `text with parameter: {{wrongParam}}`
Could help me to understand how we could gracefully handle this by some kind of fallback.

I was thinking that somebody would want to throw the error in such a case and in that case it would be not just tracking it would be handling the error.

But if you the word handle could confuse, I'll remove it.